scripts/scripts.js

```diff
@@ -166,6 +166,22 @@ function buildDocumentUrl(main) {
   }
 }
 
+function addTitlestoIconLinks(main) {
+  const spans = main.querySelectorAll('span');
+  spans.forEach((span) => {
+    const a = span.closest('a');
+    if (a) {
+      // all icons have a classlist of 'icon icon-name'
+      const iconName = span.classList[1];
+      if (iconName) {
+        a.title = iconName;
+      } else {
+        a.title = 'icon';
+      }
```

Comment:
```suggestion
      a.title = iconName ? iconName : 'icon';
```

Reply:
Suggestions welcome. It was enough to squelch the Lighthouse error.